### PR TITLE
IA-1624 IA-1602 IA-1617 IA-1629 IA-1627 Docker updates!

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/IA-1624 (gatk label should be all caps)
https://broadworkbench.atlassian.net/browse/IA-1602 (make docker images runnable locally)
https://broadworkbench.atlassian.net/browse/IA-1617 (install FISS in base image)
https://broadworkbench.atlassian.net/browse/IA-1629 (add openjdk to the base image)
https://broadworkbench.atlassian.net/browse/IA-1627 (default to gatk image in Leo)

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/src/test/resources/reference.conf [hunk 1]
```diff
@@ -1,10 +1,10 @@
-leonardo.rImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.8"
-leonardo.pythonImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.7"
-leonardo.hailImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:0.0.5"
-leonardo.gatkImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.9"
-leonardo.baseImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:0.0.6"
+leonardo.rImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.9_tmp"
```
Comment (rtitle) [inline automation/src/test/resources/reference.conf]: Will revert these `tmp`s, I just want to test them.

File: http/src/main/resources/reference.conf [hunk 2]
```diff
@@ -30,7 +30,8 @@ dataproc {
 
 image {
   welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:ed93873"
-  jupyterImage = "us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da"                  # updated 8/26/19
+  jupyterImage =  "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.10_tmp"  # updated 1/21/20
```
Comment (rtitle) [inline http/src/main/resources/reference.conf]: https://broadworkbench.atlassian.net/browse/IA-1627

**Comment by rtitle** [comment]
jenkins retest

**Review by jdcanas — APPROVED** [review]
just remove the _tmp, otherwise those will get baked into the custom dataproc image.

**Comment by rtitle** [comment]
Yeah will revert the tmps... when ready I think I will merge the terra-docker PR, which will cause new images to get pushed, then update this PR with the “correct” tags and retest.

**Pull request merged**